Source/Core/VideoBackends/D3D/main.cpp

```diff
@@ -113,7 +113,7 @@ void InitBackendInfo()
 				else if (modes[i].Quality)
```

Comment:
unless i missed something, ```DXGI_SAMPLE_DESC.Quality``` is always set to 0 at this point, so this branch could technically be removed (see ```DX11::D3D::EnumAAModes```)